package.json

```diff
@@ -44,13 +44,13 @@
     "@gnosis.pm/conditional-tokens-contracts": "^1.0.1",
     "@gnosis.pm/conditional-tokens-market-makers": "^1.8.1",
     "@gnosis.pm/truffle-nice-tools": "^1.3.1",
-    "@graphprotocol/graph-cli": "^0.18.0",
-    "@graphprotocol/graph-ts": "^0.18.1",
+    "@graphprotocol/graph-cli": "^0.19.0",
+    "@graphprotocol/graph-ts": "^0.19.0",
```

Comment:
Oh interesting, I made a conflict by merging something earlier, oops...

Should be an easy fix. I actually prefer the way it is written in this PR.

Reply:
fixed `d8c9763`